panic when importing into a table with virtual computed columns

**Describe the problem**

Importing a table with a virtual computed column causes a panic.

**To Reproduce**

```
IMPORT TABLE addresses (
    first STRING,
    last STRING,
    street STRING,
    city STRING,
    state STRING,
    zip STRING,
    first_lower STRING AS (lower(first)) VIRTUAL
) CSV DATA ('https://people.sc.fsu.edu/~jburkardt/data/csv/addresses.csv');
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x707b3cb]

goroutine 3479 [running]:
github.com/cockroachdb/cockroach/pkg/sql.NewTableDesc(0x952fd20, 0xc001f7eb38, 0x0, 0x964adc0, 0xc000ca41e8, 0xc00152c000, 0xc000874f00, 0x1d00000032, 0xc000000035, 0x0, ...)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/create_table.go:1807 +0x578b
github.com/cockroachdb/cockroach/pkg/ccl/importccl.MakeSimpleTableDescriptor(0x952fd20, 0xc001f7eb38, 0xc000747ef8, 0xc00152c000, 0xc000874f00, 0x1d00000032, 0xc000000035, 0x0, 0xc0010d5380, 0x0, ...)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/import_table_creation.go:158 +0xa85
github.com/cockroachdb/cockroach/pkg/ccl/importccl.importPlanHook.func1(0x9510260, 0xc002d2edc0, 0x0, 0x0, 0x0, 0xc002a08480, 0x0, 0x0)
        /go/src/github.com/cockroachdb/cockroach/pkg/ccl/importccl/import_stmt.go:853 +0x21e8
github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).startExec.func1(0xc000e1f200, 0x9510260, 0xc002d2edc0, 0xc002822000, 0xc000747dd8)
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:145 +0x7d
created by github.com/cockroachdb/cockroach/pkg/sql.(*hookFnNode).startExec
        /go/src/github.com/cockroachdb/cockroach/pkg/sql/planhook.go:144 +0xcc
```

**Environment:**

```
Build Tag:        v21.1.1
Build Time:       2021/05/24 15:00:00
Distribution:     CCL
Platform:         darwin amd64 (x86_64-apple-darwin19)
Go Version:       go1.15.11
C Compiler:       Clang 10.0.0
Build Commit ID:  f6630b3f524f3a5e52b330220037af01369b1ff2
Build Type:       release
(use 'cockroach-v21.1.1 version --build-tag' to display only the build tag)
```

--- a/pkg/ccl/importccl/csv_internal_test.go
+++ b/pkg/ccl/importccl/csv_internal_test.go
@@ -50,6 +50,10 @@ func TestMakeSimpleTableDescriptorErrors(t *testing.T) {
 			error: `this IMPORT format does not support foreign keys`,
 		},
 		{
+			stmt:  "create table a (i int, j int as (i + 10) virtual)",
+			error: `to import into a table with virtual computed columns, use IMPORT INTO`,
+		},
+		{
 			stmt: `create table a (
 				i int check (i > 0),
 				b int default 1,

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -4499,12 +4499,28 @@ INSERT INTO users (a, b) VALUES (1, 2), (3, 4);
 			expectedError: "to use computed columns, use IMPORT INTO",
 		},
 		{
+			into:          false,
+			name:          "import-table-csv-virtual",
+			data:          "35,23\n67,10",
+			create:        "a INT, c INT AS (a + b) VIRTUAL, b INT",
+			format:        "CSV",
+			expectedError: "to import into a table with virtual computed columns, use IMPORT INTO",
+		},
+		{
 			into:            false,
 			name:            "import-table-avro",
 			data:            avroData,
 			create:          "a INT, c INT AS (a + b) STORED, b INT",
 			format:          "AVRO",
 			expectedResults: [][]string{{"1", "3", "2"}, {"3", "7", "4"}},
+		},
+		{
+			into:          false,
+			name:          "import-table-avro-virtual",
+			data:          avroData,
+			create:        "a INT, c INT AS (a + b) VIRTUAL, b INT",
+			format:        "AVRO",
+			expectedError: "to import into a table with virtual computed columns, use IMPORT INTO",
 		},
 		{
 			into:            false,

--- a/pkg/ccl/importccl/import_table_creation.go
+++ b/pkg/ccl/importccl/import_table_creation.go
@@ -126,6 +126,11 @@ func MakeSimpleTableDescriptor(
 			*tree.UniqueConstraintTableDef:
 			// ignore
 		case *tree.ColumnTableDef:
+			if def.IsComputed() && def.IsVirtual() {
+				return nil, unimplemented.NewWithIssueDetail(56002, "import.computed",
+					"to import into a table with virtual computed columns, use IMPORT INTO")
+			}
+
 			if err := sql.SimplifySerialInColumnDefWithRowID(ctx, def, &create.Table); err != nil {
 				return nil, err
 			}

